theano/tensor/nnet/tests/test_corr.py

```diff
@@ -425,8 +425,7 @@ def test_non_contiguous(self):
 
 class TestUnsharedCorr2D(utt.InferShapeTester):
     if theano.config.mode == "FAST_COMPILE":
-        # mode = theano.compile.get_mode("FAST_RUN").excluding("conv_dnn")
-        mode = theano.compile.get_default_mode().including('gpuarray').excluding('gpu').excluding('cudnn')
+        mode = theano.compile.get_mode("FAST_RUN").excluding("conv_dnn")
```

Comment:
Doyou need to exclude conv_dnn here? Since the shared convolution tests don't do this.